recipes/python-wand/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.5.5" %}
+{% set hash_value = "c97029751f595d96ae0042aec0e26ff114e403e060ae2481124abbcca0c65ce2" %}
+
+package:
+  name: python-wand
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/emcconville/wand/releases/download/0.5.5/Wand-0.5.5.tar.gz
+  sha256: '{{ hash_value }}'
+
+build:
+  skip: true  # [win]
```

Comment:
Can you add `noarch: python`?

Reply:
You can then remove the `skip:` entry.

Reply:
Wand require  imagemagick and this is only available for  linux-64 and osx-64. That's why I included the skip: # [win]

Reply:
So you could still have it noarch and it would simply be uninstallable on windows until there is an imagemagick for win.  This is a better option overall